adapters/appnexus/appnexus.go

```diff
@@ -42,19 +42,26 @@ type KeyVal struct {
 }
 
 type appnexusParams struct {
-	PlacementId       int      `json:"placementId"`
-	InvCode           string   `json:"invCode"`
-	Member            string   `json:"member"`
-	Keywords          []KeyVal `json:"keywords"`
-	TrafficSourceCode string   `json:"trafficSourceCode"`
-	Reserve           float64  `json:"reserve"`
-	Position          string   `json:"position"`
+	LegacyPlacementId       int             `json:"placementId"`
+	LegacyInvCode           string          `json:"invCode"`
+	LegacyTrafficSourceCode string          `json:"trafficSourceCode"`
+	PlacementId             int             `json:"placement_id"`
+	InvCode                 string          `json:"inv_code"`
+	Member                  string          `json:"member"`
+	Keywords                []KeyVal        `json:"keywords"`
+	TrafficSourceCode       string          `json:"traffic_source_code"`
+	Reserve                 float64         `json:"reserve"`
+	Position                string          `json:"position"`
+	UsePmtRule              *bool           `json:"use_pmt_rule"`
+	PrivateSizes            json.RawMessage `json:"private_sizes"`
```

Comment:
I see that these are experimental so just leaving a note, not requesting any changes:

Once these become more well defined, I imagine we'd want to provide more concrete types instead of `json.RawMessage` (e.g. `[]PrivateSize`) for validation.

Reply:
Only if we actually want to do any validation on the values. Otherwise best to leave this as JSON so we save an `UnMarshall`/`Marshall` cycle on the contents.

Reply:
Makes sense. As this is an adapter it might make more sense to defer validation of fields like this to whatever service the adapter is calling out to.